main.cpp

```diff
@@ -52,86 +52,92 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  wombat
+//  action 1: the wombat sleeps
+wombat.sleep();
+//  action 2: the wombat runs
+wombat.run();
+//  action 3: the wombat poops
+wombat.poop();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  chef
+//  action 1:  the chef cooks
+chef.cook();
+//  action 2:  the chef tastes
+chef.taste();
+//  action 3:  the chef cuts
+chef.cut();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  wind
+//  action 1:  the wind howls
+wind.howl();
+//  action 2:  the wind blows
+wind.blow();
+//  action 3:  the wind stops
+wind.stop();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  water
+//  action 1:  the water pools
+water.pool();
+//  action 2:  the water frezzes
+water.freeze();
+//  action 3:  the water boils
+water.boil();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  bird
+//  action 1:  the bird flies
+bird.fly();
+//  action 2:  the bird sings
+bird.sing();
+//  action 3:  the bird makes a nest  
+bird.makeNest();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: lush
+//  action 1:  the lush drinks
+lush.drink();
+//  action 2:  the lush talks
+lush.talk();
+//  action 3:  the lush regrets
+lush.regret();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: river
+//  action 1:  the river flows
+river.flow();
+//  action 2: the river floods
+river.flood();
+//  action 3:  the river freezes
+river.freeze();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  toddler  
+//  action 1:  the toddler laughs
+toddler.laugh();
+//  action 2:  the toddler naps
+toddler.nap();
+//  action 3:  the toddler cries
+toddler.cry();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.